tools/buildTools/rbcd

```diff
@@ -0,0 +1,85 @@
+function _rc_list_locations {
```

Comment:
Some code or internal documentation would be great. It doesn't need to be too extensive. Maybe you can add an explanation of the function, input, and output? Thank you in advance.

Reply:
yes sure, will add some comments soon.

Reply:
I have added the comments.